src/Sulu/Bundle/SearchBundle/Tests/Functional/BaseTestCase.php

```diff
@@ -24,9 +24,6 @@ class BaseTestCase extends SuluTestCase
     public function setUp()
     {
         $this->initPhpcr();
-        $fs = new Filesystem;
-
-        $fs->remove(__DIR__ . '/../app/data');
```

Comment:
I delete before. It gives you the chance to debug, if you delete after you need to do things like `die()` to inspect the state of the database.

But tbh I am not sure why I removed this, I just noticed that it did no harm. Will verify.